src/PointwiseFunctions/GeneralRelativity/WeylPropagating.hpp

```diff
@@ -0,0 +1,72 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace gr {
+// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the propagating modes of the Weyl tensor
+ *
+ * \details The Weyl tensor evolution system in vacuum has six characteristic
+ * fields, of which two (\f$ U^{8\pm}\f$) are proportional to the
+ * Newman-Penrose components of the Weyl tensor \f$\Psi_4\f$ and \f$\Psi_0\f$.
+ * These represent the true gravitational-wave degrees of freedom, and
+ * can be written down in terms of \f$3+1\f$ quantities as
+ * \cite Kidder2004rw (see Eq. 75):
+ * \f{align}
+ * U^{8\pm}_{ij} &= \left(P^{k}_i P^{l}_j - \frac{1}{2} P_{ij} P^{kl}\right)
+ *          \left(R_{kl} + K K_{kl} - K_k^m K_{ml}
+ *                \mp n^m \nabla_m K_{kl} \pm n^m \nabla_{(k}K_{l)m}\right),\\
+ *  &= \left(P^{k}_i P^{l}_j - \frac{1}{2} P_{ij} P^{kl}\right)
+ * \left(E_{kl} \mp n^m \nabla_m K_{kl} \pm n^m \nabla_{(k}K_{l)m}\right),
+ * \f}
+ * where \f$R_{ij}\f$ is the spatial Ricci tensor, \f$K_{ij}\f$ is the
+ * extrinsic curvature, \f$K\f$ is the trace of \f$K_{ij}\f$, \f$E_{ij}\f$ is
+ * the electric part of the Weyl tensor in vacuum, \f$n^i\f$ is a spatial unit
```

Comment:
`is the outward directed unit normal vector to the interface` is the wording used for n^i in GH/BC/Upwind so I think we should be consistent

Reply:
reworded